app/views/admin/loans/show.html.slim

```diff
@@ -1,4 +1,4 @@
-- content_for(:title, @loan.display_name)
+- content_for(:title, "#{@loan.id}, #{@loan.display_name}")
```

Comment:
I like it. Nice and simple. Can you please change the comma to a colon though, and put a #. So like 

#5674: Project with Sassy Pants

Thanks!